csharp/src/Drivers/Apache/Hive2/HiveServer2Connection.cs

```diff
@@ -75,6 +75,8 @@ internal async Task OpenAsync()
 
         protected internal HiveServer2TlsOption TlsOptions { get; set; } = HiveServer2TlsOption.Empty;
 
+        protected internal int RequestTimeout { get; set; } = 30000;
```

Comment:
Slightly pedantic, but should this specify that it is an HTTP timeout since we may support other ways of connecting?

Reply:
Updated properties and options to indicate its use with the HTTP type connection.